AmazonChimeSDK/AmazonChimeSDK/audiovideo/video/backgroundfilter/backgroundblur/BackgroundBlurConfiguration.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  BackgroundBlurConfiguration.swift
+//  AmazonChimeSDK
+//
+//  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//  SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+@objcMembers public class BackgroundBlurConfiguration {
+    var blurStrength: BackgroundBlurStrength
```

Comment:
Looks like we are not changing those variables after init, use `let` instead of `var`.

Reply:
done